src/indexApi.ts

```diff
@@ -0,0 +1,33 @@
+// BASE SETUP
```

Comment:
I know some prefer using a standard of index.js so that it's clear where to start when looking at the code, but as long as you're considerate with the rest of your file naming scheme I think it should be fine.

Reply:
Thanks Lex. My plan is to have two 'index' files: `indexApi` and `indexEngine`.